python_modules/dagster/dagster/_core/definitions/asset_daemon_context.py

```diff
@@ -132,6 +129,10 @@ def target_asset_keys_and_parents(self) -> AbstractSet[AssetKey]:
             for parent in self.asset_graph.get_parents(asset_key)
         } | self.target_asset_keys
 
+    @property
+    def respect_materialization_data_versions(self) -> bool:
+        return self._respect_materialization_data_versions
+
     def get_implicit_auto_materialize_policy(
```

Comment:
Not related to this PR, but can we get rid of this function now that the reconciliation sensor is removed?

Reply:
I believe @johannkm had a PR that did so, but it got reverted -- I forget why exactly, but yeah in principle there's no reason it needs to exist other than to keep the existing tests running as usual